osmo-test-5/osmosis.zone_chains.json

```diff
@@ -6,178 +6,126 @@
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go"]
     },
     {
       "chain_name": "composabletestnet",
       "rpc": "https://rpc-banksy.notional.ventures",
       "rest": "https://api-banksy.notional.ventures",
       "explorer_tx_url": "https://explorer.nodexcapital.com/composable-3/tx/${txHash}",
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go"]
     },
     {
       "chain_name": "persistencetestnet2",
       "rpc": "https://rpc.testnet2.persistence.one",
       "rest": "https://rest.testnet2.persistence.one",
       "explorer_tx_url": "https://mintscan.io/persistence-testnet/tx/${txHash}",
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go"]
     },
     {
       "chain_name": "xiontestnet",
       "rpc": "https://rpc.xion-testnet-1.burnt.com:443",
       "rest": "https://api.xion-testnet-1.burnt.com",
       "explorer_tx_url": "https://pingpub.xion-testnet-1.burnt.com/xion-testnet-1/tx/${txHash}",
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go"]
     },
     {
       "chain_name": "sagatestnet",
       "rpc": "https://testnet-ssc.sagarpc.io/",
       "rest": "https://testnet-ssc-lcd.sagarpc.io/",
       "explorer_tx_url": "https://www.mintscan.io/saga-testnet/tx/${txHash}",
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go"]
     },
     {
       "chain_name": "impacthubtestnet",
       "rpc": "https://rpc.testnet.ixo.earth",
       "rest": "https://testnet.ixo.earth/rest",
       "explorer_tx_url": "https://blockscan-pandora.ixo.earth/transactions/${txHash}",
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go"]
     },
     {
       "chain_name": "archwaytestnet",
       "rpc": "https://rpc.constantine.archway.io:443",
       "rest": "https://api.constantine.archway.io:443",
       "explorer_tx_url": "https://www.mintscan.io/archway-testnet/tx/${txHash}",
-      "keplr_features": [
-        "ibc-transfer",
-        "ibc-go",
-        "cosmwasm"
-      ]
+      "keplr_features": ["ibc-transfer", "ibc-go", "cosmwasm"]
     },
     {
       "chain_name": "symphonytestnet",
       "rpc": "https://symphony-rpc.kleomedes.network",
       "rest": "https://symphony-api.kleomedes.network",
       "explorer_tx_url": "https://testnet.ping.pub/symphony/tx/${txHash}",
-      "keplr_features": [
-        "ibc-go",
-        "ibc-transfer",
-        "cosmwasm",
-        "wasmd_0.24+"
-      ]
+      "keplr_features": ["ibc-go", "ibc-transfer", "cosmwasm", "wasmd_0.24+"]
+    },
+    {
+      "chain_name": "kimatestnet",
```

Comment:
This chain doesn't yet have fees defined at the Chain Registry

Reply:
So you mean I have to add
![Screenshot 2024-09-09 at 12 04 52 PM](https://github.com/user-attachments/assets/733d36ec-9b36-487d-95fd-90db8e091b0d)
I guess we have. it's same structure, other chains

Reply:
It needs to low, average, and high gas prices as well